readme.md

```diff
@@ -323,13 +337,13 @@ If we run these commands, we're dropping our entire database and creating a bran
 * Why is this potentially super dangerous?
 
 Another (wrong) way we can do this is by using `rails db:rollback`.
```

Comment:
I'd kill the word `wrong` here. It's totally fine to do this in development, as is stated later in the lesson. This opens up questions to why teach if wrong. But also, not really wrong. Maybe move up the language about this being ok in development not production.